tfhe/docs/getting_started/security_and_cryptography.md

```diff
@@ -79,43 +54,40 @@ In FHE, there are two types of operations that can be applied to ciphertexts:
 
 In FHE, the noise must be tracked and managed in order to guarantee the correctness of the computation.
 
-Bootstrapping operations are used across the computation to decrease the noise in the ciphertexts, preventing it from tampering the message. The rest of the operations are called leveled because they do not need bootstrapping operations and thus are most of the time really fast.
+Bootstrapping operations are used across the computation to decrease the noise in the ciphertexts, preventing it from tampering the message. The rest of the operations are called leveled because they do not need bootstrapping operations and, thus, are usually really fast.
 
 The following sections explain the concept of noise and padding in ciphertexts.
 
-### Noise
+### Noise.
```

Comment:
why is there a column here when it was removed elsewher I believe ? If it's a gitbook formatting thing, then never mind